puppet/modules/socorro/manifests/init.pp

```diff
@@ -62,11 +62,9 @@
   }
 
   package {
-    [
-      'apache-maven',
-    ]:
-    ensure  => latest,
-    require => Yumrepo['dchen'];
+    'apache-maven':
+      ensure  => latest,
+      require => Yumrepo['dchen'];
```

Comment:
I didn't want to alter functionality in any way with this bug - this is purely stylistic.